_pytest/compat.py

```diff
@@ -253,6 +253,29 @@ def safe_str(v):
             return v.encode('ascii', errors)
 
 
+COLLECT_FAKEMODULE_ATTRIBUTES = (
+    'Collector',
+    'Module',
+    'Generator',
+    'Function',
+    'Instance',
+    'Session',
+    'Item',
+    'Class',
+    'File',
+    '_fillfuncargs',
+)
+
+
+def _setup_collect_fakemodule():
```

Comment:
@nicoddemus that breaks `python -m pytest` on python2.6 - else i could just make it a file, move the main hack and be happy